openapi_tester/schema_tester.py

```diff
@@ -253,6 +257,50 @@ def _validate_openapi_type(self, schema_section: dict, data: str) -> Union[Optio
                 f"but received {type(data).__name__}."
             )
 
+    @staticmethod
+    def _validate_multiple_of(schema_section: dict, data: Any) -> Union[Optional[str], bool]:
+        multiple = schema_section.get("multipleOf")
+
+        if multiple is None:
+            return False
+
+        if not data % multiple == 0:
+            return f"The response value {data} should be a multiple of {multiple}"
+
+    @staticmethod
+    def _validate_min_and_max(schema_section: dict, data: Any) -> Union[Optional[str], bool]:
+        minimum = schema_section.get("minimum")
+        if minimum is not None:
+            exclusive_minimum = schema_section.get("exclusiveMinimum")
+            if not exclusive_minimum:
+                if data < minimum:
+                    return f"The response value {data} exceeds the minimum allowed value of {minimum}"
+            else:
+                if data <= minimum:
+                    return f"The response value {data} exceeds the minimum allowed value of {minimum + 1}"
+
+        maximum = schema_section.get("maximum")
+        if maximum is not None:
+            exclusive_maximum = schema_section.get("exclusiveMaximum")
+            if not exclusive_maximum:
+                if data > maximum:
```

Comment:
can be combined with the containing if clause. 

I suggest adding pylint for this.

Reply:
Yeah perhaps - this seems easier to read though, no? This way the comparisons are aligned